dev-helpers/dev-helper-initializer.js

```diff
@@ -1,11 +1,36 @@
 /* eslint-disable no-undef */
-window.onload = function() {
+window.onload = function () {
   window["RestImportBundle"] = window["rest-import-ui-bundle"]
   window["RestImportStandalonePreset"] = window["rest-import-ui-standalone-preset"]
   // Build a system
   const ui = RestImportBundle({
     dom_id: "#rest-import-ui",
-    language:'zh',
+    language: 'zh',
```

Comment:
@Ashokkumar1798 let the language be EN by default

Reply:
Changed language to 'en'